app/javascript/dashboard/assets/scss/_rtl.scss

```
@@ -312,8 +322,16 @@
   }
 
   // Labels
-  .label {
-    direction: initial;
+  .label-wrap .label {
+    .label-color-dot {
+      margin-left: var(--space-smaller);
+      margin-right: 0;
+    }
+
+    .label-close--button {
+      margin-left: 0;
+      margin-right: var(--space-smaller);
+    }
```

We should probably just make `.label` a `flex` and give it a gap of `--space-smaller

```suggestion
.label-wrap .label {
    display: flex;
    gap: var(--space-smaller);
}
```

LMK if this works for the use case

---

This looks clean 👍

---

@scmmishra @nithindavid Updated